mobile/src/main/java/com/gelakinetic/mtgfam/fragments/ResultListFragment.java

```diff
@@ -440,8 +440,9 @@ private void showDialog() throws IllegalStateException {
      * Called when the sort dialog closes. Sort the cards according to the new options.
      *
      * @param orderByStr The sort order string
+     * @param dontSort   ignored
      */
-    public void receiveSortOrder(String orderByStr) {
+    public void receiveSortOrder(String orderByStr, boolean dontSort) {
```

Comment:
Actually, you're ordering by "Order Added". Consider including this in the orderByStr.